solutions/ojuz-knapsack.mdx

```diff
@@ -0,0 +1,171 @@
+---
+id: ojuz-knapsack
+source: Singapore NOI
+title: knapsack
```

Comment:
Make the k uppercase here and change the problem id and it should be good to go

Reply:
problem id? what should it be?

Reply:
Like `noi-18-knapsack` or something